addons/mail/static/src/js/form_controller.js

```diff
@@ -33,6 +34,23 @@ FormController.include({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * Open the composer and scroll it into view.
+     * @private
+     * @param {OdooEvent} ev
+     * @param {String} ev.data.messageType 'note' or 'message'
+     */
+    _onChatterFocus: async function (ev) {
+        const chatter = this.renderer.chatter;
+        if (chatter) {
+            if (ev.data.messageType === 'note') {
+                await chatter._onOpenComposerNote();
```

Comment:
This method already exists on the chatter widget; it's called `_openComposer`; I'll make it public and call it from my handler in the form controller.

Reply:
But i'm not a fan of modifying other's people modules, and it doesn't bother me that much that it's a private method that we're calling, since we're in the same module anyway; but why not. I'll just notify aku.